src/app/filament/change-filament/change-filament.component.html

```diff
@@ -1,6 +1,6 @@
 <ng-container *ngIf="!usingM600(); else M600Change">
   <span class="filament-heading" i18n="@@load-new-filament">load new filament</span>
-  <span class="change-filament__info" i18n="@@load-new-filament-message">only put a little filament in, I'll pull in the rest.</span>
+  <span class="change-filament__info" i18n="@@load-new-filament-message">push the filament until feeling a slight pressure and press done</span>
```

Comment:
this might work for your specific printer model, but isn't applicable to every printer. So i would prefer keeping the original text here or something more general.